### PR TITLE
Spanish translation (WIP)

### PR DESCRIPTION
**Pull request by Cantejito**
I'm going to keep updating it on my free time. No ETA, but should not take long.

Is the first time that I'm doing this, so feel free to tell me if something can be done better.

### CODE REVIEW

File: Dialogs/Dialogs.es.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "OtherInstance": "Otra instancia de Fan Control estaba en uso en el usuario \"{0}\". Se ha cerrado para iniciar esta nueva estancia.",
+  "PaypalDialogPart1": "Mensaje del desarrollador de Fan Control",
+  "PaypalDialogPart2": "¡Hola! Aquí Rémi.",
+  "PaypalDialogPart3a": "¡Veo que has hecho unas curvas, impresionante! 🚀🙌",
+  "PaypalDialogPart3b": "¡Aún no has creado curvas para tus ventiladores! ¡Haz clic en el botón grande de abajo a la derecha con el símbolo + para empezar! 🚀🙌",
+  "PaypalDialogPart4": "El programa está siendo continuamente desarrollado para obtener mejor rendimiento, menos errores, más funciones y una mejor experiencia de usuario. Eso conlleva tiempo y esfuerzo.",
+  "PaypalDialogPart5": "Fan Control es gratis de usar, pero si te gusta lo suficiente como para que forme parte de tu sistema, considera compartirlo, contribuir en Paypal o patrocinarme en GitHub. ¡Estaría muy agradecido! 😊",
+  "PaypalDialogPart6": "Si tienes problemas con el programa o simplemente quieres compartir tus ideas, tienes un email en la sección "Información" en la parte izquierda o puedes ir directamente a la página de GitHub.",
```
Comment (Rem0o) [inline Dialogs/Dialogs.es.json:34]: Double quotes need to be escaped.   \"Información\"

**Comment by Cantejito** [comment]
All files translated and ready to merge.

As I said, I will keep updating it for possible grammar fixes and consistency.

**Comment by Rem0o** [comment]
> All files translated and ready to merge.
> 
> As I said, I will keep updating it for possible grammar fixes and consistency.

See my comment on Dialog.es.json, there is a format error with the double quotes.

**Comment by Cantejito** [comment]
> > All files translated and ready to merge.
> > As I said, I will keep updating it for possible grammar fixes and consistency.
> 
> See my comment on Dialog.es.json, there is a format error with the double quotes.

It was fixed on that commit. I forgot to mention it on the comment.

**Pull request merged**